css/templates/e-commerce/page.css

```diff
@@ -0,0 +1,216 @@
+@import '../../core';
+@import 'page-vars';
+@import '../../../components/inputs/inputs';
+@import '../../../components/social-follow/social-follow';
+
+body {
+  font-size: var(--narrow-body-font-size);
+}
+
+@media (--breakpoint-md) {
+  body {
+    font-size: var(--wide-body-font-size);
+  }
+}
+
+main {
+  max-width: none;
+}
+
+
+/* TYPOGRAPHY */
+.h3 {
```

Comment:
I'm not sure this would be any more efficient:

```
.ampstart-title-sm {
  font-size: var(--h3);
  line-height: var(--line-height-4);
  letter-spacing: 0.06rem;
}

.h3 {
  color: var(--color-font-title);
  line-height: normal;
  padding-bottom: var(--space-1);
}
```

We would need to override the line height and letter spacing and then add padding and colour. This results in much larger CSS.